tb/cli.py

```diff
@@ -1,14 +1,16 @@
-"""the adjust tinderbox
+import logging
+import pydoc
+import sys
 
-Usage:
-  tinderbox
-  tinderbox (-h | --help)
-  tinderbox --version
+import docopt
 
-Options:
-    -h|--help     display help.
-    --version     display version.
-"""
+import tb.backend as tbb
+
+
+logging.basicConfig(
+    level=logging.INFO, format="[%(asctime)s][%(name)s][%(levelname)s]: %(message)s"
+)
```

Comment:
This configuration is done each time in every module. We could deduplicate it.

Reply:
I've been looking for a way to refactor this code ... to no avail :(